Decks/Source/LimitedDeck.cpp

```diff
@@ -0,0 +1,47 @@
+#include "LimitedDeck.hpp"
+
+HandDeck::HandDeck(int maxSize)
+:_maxSize(maxSize)
+{
+
+}
+
+TableDeck::TableDeck(int maxSize)
+:_maxSize(maxSize)
+{
+
+}
+
+void HandDeck::drawCard(Deck &deck)
+{
+  if(_size >= _maxSize)

```

Comment:
_size brzmi prawie tak samo jak _maxSize, zakładam, że to aktualna ilość kart, stąd dałbym raczej coś w style currentNumberOfCards (albo coś lepszego, bo to też nie siedzi mi)